Revisions to 'Texel' glossary entry (#18877)
mdn/content: files/en-us/glossary/texel/index.md

  - Texel
  - Texture
---

> In 3D graphics, a **Texel** is a single pixel within a texture. **Textures** are images that are presented on a polygon's surface within a 3D rendered image. A texture is characterized by a collection of texels, as how an image is characterized by a collection of pixels. 

Reviewer: ```suggestion
In 3D graphics, a **Texel** is a single pixel within a texture. **Textures** are images that are presented on a polygon's surface within a 3D rendered image. A texture is characterized by a collection of texels, similar to how an image is characterized by a collection of pixels. 
```

Author: This change was bundled with another suggestion below as part of https://github.com/mdn/content/pull/18877/commits/9cc28a2fdb8ac72aaaec0a08887f14f957848c4e

> In 3D graphics, a **Texel** is a single pixel within a texture. **Textures** are images that are presented on a polygon's surface within a 3D rendered image. A texture is characterized by a collection of texels, as how an image is characterized by a collection of pixels. 

Reviewer 2: Let's not use a capital letter here (Estelle's suggestion included):
```suggestion
In 3D graphics, a **texel** is a single pixel within a texture. _Textures_ are images presented on a polygon's surface within a 3D rendered image. A texture is characterized by a collection of texels, similar to how an image is characterized by a collection of pixels. 
```

> In 3D graphics, a **Texel** is a single pixel within a texture.  **Textures** are images that are presented on a polygon's surface within a 3D rendered image.  A texture is characterized by a collection of texels, as how an image is characterized by a collection of pixels. 

Reviewer 3: Not that it matters much right now, but our style is prevalently single space after full stop (or a line break, if you wish)

Author: Thanks for letting me know!  Addressed in https://github.com/mdn/content/pull/18877/commits/eb4d1aae2c3dcd78e27d8595b153a0cafd0267db

> A pixel in a raster image file is a series of bits containing color and possibly opacity data, which maps to display pixels on an output device such as a computer monitor. When a pixel belongs to an image used as a texture resource, it is called a 'texture pixel' or shortened to 'texel'. Instead of mapping directly to screen pixels, a texel's data is mapped to a location in the coordinate space of the 3D object being modelled. Textures may be used to convey color, but also other surface qualities such as depth or reflectivity. Multiple textures may be layered to create complex surface overlays.

Reviewer: ```suggestion
A pixel in a raster image file is a series of bits containing color data, and sometimes opacity data, which maps to display pixels on an output device such as a computer monitor. When a pixel belongs to an image used as a texture resource, it is called a 'texture pixel' or shortened to 'texel'. Instead of mapping directly to screen pixels, a texel's data is mapped to a location in the coordinate space of the 3D object being modeled. Textures can be used to convey color and other surface qualities such as depth and reflectivity. Multiple textures may be layered to create complex surface overlays.
```

> The process of mapping the appropriate texels to their corresponding points on a polygon is called **texture mapping**, which is a stage of the process of rendering a 3D image for display. When the source texel grid and destination pixel grid do not align, further **texture filtering** is applied to smooth the resultant texture-mapped pixels (texture _magnification_ or _minification_). The final output of the rendering process is a flattened 2D projection of the 3D model, where the texture has been 'wrapped' around the model.

Reviewer: ```suggestion
The process of mapping the appropriate texels to their corresponding points on a polygon is called **texture mapping**.  Texture mapping is a stage of the process of rendering a 3D image for display. When the source texel grid and destination pixel grid do not align, further **texture filtering** is applied to smooth the resultant texture-mapped pixels (texture _magnification_ or _minification_). The final output of the rendering process is a flattened 2D projection of the 3D model, where the texture has been 'wrapped' around the model.
```
